smpte.js

```diff
@@ -686,6 +703,14 @@ function insertConformance(docMetadata) {
     interoperability information.</p>
     `;
 
+  } else if (docMetadata.pubType === smpte.ER_PUBTYPE) {
+
+    sec.innerHTML = `
+    <h2>Conformance</h2>
+    <p>This Engineering Guideline is purely informative and meant to information to the 
```

Comment:
```suggestion
    <p>This Engineering Guideline is meant to provide information to the 
```

Reply:
Updated as suggested in https://github.com/SMPTE/html-pub/pull/315/commits/4276f14ad73bb02f932bac5d464ec691179dd1ab